src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/TypePositionInfo.cs

```diff
@@ -47,6 +47,17 @@ public sealed record TypePositionInfo(ManagedTypeInfo ManagedType, MarshallingIn
     {
         public const int UnsetIndex = int.MinValue;
         public const int ReturnIndex = UnsetIndex + 1;
+        public const int ExceptionIndex = UnsetIndex + 2;
```

Comment:
I might be missing it - where does the generator create a `TypePositionInfo` using `ExceptionIndex` or `IncrementalStubGenerationContext.ExceptionMarshallingInfo`?